input/images/example-authorization-jwks-and-signatures.ipynb

```diff
@@ -6,7 +6,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# To create a markdown of this notebook, run: jupyter nbconvert --to markdown authorization-example-jwks-and-signatures.ipynb\n",
+    "# To create a markdown of this notebook, run: jupyter nbconvert --to markdown example-authorization-jwks-and-signatures.ipynb\n",
```

Comment:
I think the changes to these python examples aren't related to Task 38664. Could you submit these changes in a separate PR?

Reply:
Sure, if they are needed. Are they?

Reply:
The examples are needed in the spec, but they're not related to this PR. I see there was another PR (https://github.com/HL7/smart-app-launch/pull/390) to introduce changes, and an outstanding question from @Healthedata1 on that PR.

Reply:
Likely, it was a leftover from the other PR. Do I understand correctly, that we can remove them?

Reply:
We cannot remove the examples from the spec. This PR should not change the examples, since they're not mentioned in the resolution of FHIR-38664